internal/pkg/runtime/engines/singularity/plugins.go

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+// +build cni_plugins
```

Comment:
To avoid confusion, maybe add a comment mentioning this is a hack for `go mod` and `cni_plugins` build tag is not intended to be used during build

Reply:
Good point, I'll update this.